bigbluebutton-tests/playwright/package.json

```diff
@@ -4,7 +4,8 @@
     "test:filter": "npx playwright test -g",
     "test:headed": "npx playwright test --headed",
     "test:debug": "npx playwright test --debug -g",
-    "test-ci": "export CI='true' && npx playwright test --project=chromium --grep @ci"
+    "test-ci": "export CI='true' && npx playwright test --project=chromium --grep @ci",
```

Comment:
Minor suggestion: since `test-ci` only runs Chromium, maybe it's better to rename it to something like `test-chromium-ci` (with similar adjustments to `automated-tests.yml`) to avoid naming confusion. If at some point in the future we'll run all browsers by default, we'll be able to create `test-ci` for it.

Reply:
Will anybody notice if `test-ci` vanishes?

Reply:
As far as I can see, it's only used in `.github/workflows/automated-tests.yml`.

Reply:
commit pushed (and pushed on #15577)

Reply:
Thanks